types/gulp-stylelint/package.json

```
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "stylelint": "^14.5.1"
```

is the version correct?
https://github.com/olegskl/gulp-stylelint/blob/master/package.json#L35
```suggestion
        "stylelint": "^13.0.0"
```

---

You are right. I wanted to use the package with a different npm package. I updated that to use the correct npm package.

---

This is still v14 in the latest version, while the gulp-stylelint seems to imply that they only test with stylelint v13; maybe this has to be a `>=13` or something?

---

package is strict in peer dep and in dev deps. We are trying to avoid here adding another complexity with versions being different for dt types and native package (if possible ofc)

---

Oops, no, this is for the fork, which sets v14.0.1 like this PR currently does: https://unpkg.com/browse/@ronilaukkarinen/gulp-stylelint@14.0.5/package.json

---

Oh, cool, my fault. so, no problem here :)